source/_integrations/binary_sensor.xiaomi_aqara.markdown

```diff
@@ -70,7 +70,7 @@ The requirement is that you have setup the [`xiaomi aqara` integration](/integra
       target:
         entity_id: light.gateway_light_34ce00xxxx11
     - service: automation.turn_off
-      data:
+      target:
         entity_id: automation.Motion_off
```

Comment:
Out of scope, but this should be `automation.motion_off` right?

Reply:
Yeah, I've spotted more of these occurrences. I think I'll just need to craft a regex based on the entity validation regex that finds all these.